src/index.js

```diff
@@ -19,24 +19,44 @@ const ceParams = {
     "UsageQuantity"
   ],
   GroupBy: [{
-    Key: 'AZ',
+    Key: 'REGION',
```

Comment:
@mrchrisadams I couldn't find data on specific AZs, only regional data. If you have more granular data from AWS based on specific AZ happy to switch this back.

Reply:
Hmm… I think you're right in this case.

I haven't been able to find more granular data than this about AWS myself either,and I don't think it brings up more useful information either. Let's drop it for now 👍